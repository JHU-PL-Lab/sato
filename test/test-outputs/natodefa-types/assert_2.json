[
  {
    "err_errors": [
      [
        "Error_binop",
        {
          "err_binop_left_aliases": [],
          "err_binop_right_aliases": [],
          "err_binop_left_val": "true",
          "err_binop_right_val": "true",
          "err_binop_operation": "true <> true"
        }
      ]
    ],
    "err_input_seq": [],
    "err_location": "assert (not true)",
    "err_steps": 145
  }
]