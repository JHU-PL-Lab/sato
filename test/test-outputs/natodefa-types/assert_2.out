Type errors for:
- Input sequence : []
- Found at expr  : assert (not true)
--------------------
* Constraint : true <> true
1 errors found. No further control flows exist.
